packages/inter-protocol/src/contractSupport.js

```diff
@@ -158,3 +165,37 @@ export const makeEphemeraProvider = init => {
   };
 };
 harden(makeEphemeraProvider);
+
+export const provideEmptySeat = (zcf, baggage, name) => {
+  return provide(baggage, name, () => zcf.makeEmptySeatKit().zcfSeat);
+};
+harden(provideEmptySeat);
+
+/**
+ * For making singletons, so that each baggage carries a separate kind definition (albeit of the definer)
+ *
+ * @param {*} baggage
+ * @param {*} category
+ * @returns
```

Comment:
```suggestion
 * @param {import('@agoric/vat-data').Baggage} baggage
 * @param {string} category
 * @returns {import('@agoric/vat-data').Baggage}
```

Reply:
thanks I should have included that before R4R